lib/fileio/importData.cpp

```diff
@@ -0,0 +1,146 @@
+//
```

Comment:
New file containing all the import data, such as root prim path, stage pop mask, variants, etc.